kobuki/e964bbb/e964bbb.bug

```diff
@@ -12,28 +12,48 @@ classification: "CWE-682: Incorrect Calculation #PHYSICAL"
 keywords: ['differential drive', 'velocity', 'driver', 'movement']
 system: kobuki
 severity: error
-links:
+links: []
+failure-codes:
+  - PROGRAMMING:CALCULATIONS
+fault-codes:
+  - SYSTEM:UNINTENDED-BEHAVIOUR
+  - SYSTEM:MOTION
 bug:
   phase: runtime
   specificity: robotics-specific
   architectural-location: application-specific code
   application: mobile robot
-  task: differential drive
+  task: locomotion
   subsystem: driver
-  package: yujinrobot/kobuki/kobuki_driver
-  languages: null
+  package:
+    - yujinrobot/kobuki/kobuki_driver
+  languages:
+    - C++
   detected-by: developer
   reported-by: contributor
   issue: https://github.com/yujinrobot/kobuki/issues/227
   time-reported: 2013-02-22T09:09:21Z
   reproducibility: always
-  trace:
+  trace: N/A
 fix:
   repo: https://github.com/yujinrobot/kobuki
   hash: e964bbb8700fb1a9b95c0cfe5a44d43321294d4f
-  pull-request: null
-  license: ['BSD']
-  fix-in: ['kobuki_driver/src/driver/diff_drive.cpp']
+  pull-request: N/A
```

Comment:
Just a question: we don't use `null` any more for this then? Or was that too ambiguous (as in: could also mean we just haven't bothered to look for one)?

Reply:
I am not sure in general, but for this specific case, the commit was not part of any PR. So `N/A` might be more suitable here.